### ISSUE TITLE
Dynamically viewbox

### ISSUE THREAD

**Issue by whmk-m** [egoist/vue-content-loader#316]
When I want to dynamically change the value of the viewbox, 

I find that the computedviewbox in the component contentloader maintains the original value, 

resulting in failure to respond to the latest value

**Comment by LeBenLeBen** [comment]
Indeed, I see where the issue comes from, I should be able to release a fix this week.

**Comment by whmk-m** [comment]
Wait for your good news.

**Comment by LeBenLeBen** [comment]
Fix available in 2.0.1
